utilities/options/options_util_test.cc

```diff
@@ -94,6 +94,39 @@ TEST_F(OptionsUtilTest, SaveAndLoad) {
   }
 }
 
+TEST_F(OptionsUtilTest, SaveAndLoadWithCacheCheck) {
+  //creating db
+  DBOptions db_opt;
+  db_opt.create_if_missing = true;
+
+  // initialize BlockBasedTableOptions
+  std::shared_ptr<Cache> cache = NewLRUCache(1 * 1024 * 1024 * 1024);
+  BlockBasedTableOptions bbt_opts;
+  bbt_opts.block_size = 32 * 1024;
+  //saving cf options
+  std::vector<ColumnFamilyOptions> cf_opts;
+  ColumnFamilyOptions cf_opt_sample = ColumnFamilyOptions();
```

Comment:
do you mind updating the test to mimic the real life scenario that **multiple** column families are sharing the same block cache?